bitnami/postgresql-ha/templates/postgresql/hooks-scripts-configmap.yaml

```diff
@@ -109,8 +109,7 @@ data:
     TMP_FIRST_READY_FILE_TS="/tmp/ts-first-ready.mark"
     TMP_DELAY_APPLIED_FILE="/tmp/delay-applied.mark"
 
-
-    DB_CHECK_RESULT=$(echo "SELECT 1" | postgresql_execute "$POSTGRESQL_DATABASE" "$POSTGRESQL_USERNAME" "$POSTGRESQL_PASSWORD" "-h 127.0.0.1 -tA" || "command failed")
```

Comment:
Any idea why postgresql_execute is not available? It should be loaded from `. /opt/bitnami/scripts/libpostgresql.sh`...

Reply:
Ok let me check for it.

Reply:
Hi @DevHwan, any updates on this? If not, please let us know and we will try to investigate it on our side.

Reply:
Sorry, I did forgot to submit the review..